File: C/EX36.c
#include <stdio.h>
 
int main(int argc, char *argv[]){
  
  int numeros[20], T, a, multiplicados[20];
  
  printf("Digite os 20 numeros: ");
  for(int i=0; i<20; i++)
  {
    scanf("%i", &numeros[i]);
  }

  printf("\nDigite a constante de multiplicao:");
  scanf ("%i", &a);

  printf("\nOs resultados foram:\n ");
  for(int i=0; i<20; i++)
  {
      T=numeros[i]*a;
      printf("%i*%i=%i\n", numeros[i], a, T);
      multiplicados[i]=T;
      
  }
  printf("\nOs numeros digitados foram:\n ");
  for(int i=0; i<20; i++)
  {
      printf("%i\n", numeros[i]);
  }

  printf("\nOs resultados foram:\n ");
  for(int i=0; i<20; i++){
      printf("%i\n", multiplicados[i]);
  }

  return 0;
}

File: Python/Ex36.py
numeros = []
multiplicados = []
 
for i in range(0, 10, 1):
    multiplicados.append(10)


for i in range(0, 10, 1):
    x = int(input('Digite um numero: '))
    numeros.append(x)


a = float(input("Digite a constante de multiplicação: "))

print ("Os resultados foram: ")
for i in range (0, 10, 1):
    T= numeros[i]*a
    print(f'{numeros[i]} X {a} = {T}')
    multiplicados [i]=T

print ("Os números digitados foram:")
for i in range (0, 10, 1):
    print (numeros[i])

print ("Os resultados foram: ")
for i in range (0, 10, 1):
    print (multiplicados[i])